src/main/java/mate/academy/internetshop/controller/AddBucketItemController.java

```diff
@@ -0,0 +1,41 @@
+package mate.academy.internetshop.controller;
+
+import mate.academy.internetshop.annotation.Inject;
+import mate.academy.internetshop.model.Bucket;
+import mate.academy.internetshop.service.BucketService;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class AddBucketItemController extends HttpServlet {
+    private static final Long userId = 0L;
+    private static final Long userBucketId = 0L;
+
+    @Inject
+    static BucketService bucketService;
+
+    Bucket userBucket = new Bucket(userId);
```

Comment:
we should remove this logic from here
better create controller (for exml: /createData) and add all test data there